src/main/java/com/celements/web/plugin/cmd/PageLayoutCommand.java

```diff
@@ -414,7 +415,10 @@ private SpaceReference decideLocalOrCentral(SpaceReference layoutSpaceRef) {
         layoutSpaceRef = null;
       }
     }
-    return layoutSpaceRef;
+    if (!checkLayoutAccess(layoutSpaceRef)) {
```

Comment:
this is a new functionality (which I like very much 👍 ) Yet, a apropriate unit test is missing. Maybe making decideLocalOrCentral as rename resolveValidLayoutSpace public moreover asks for some unit tests?